x-pack/metricbeat/module/aws/sqs/_meta/data.json

```diff
@@ -0,0 +1,50 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "aws": {
+        "sqs": {
+            "empty_receives": {
+                "count": 0
```

Comment:
I wonder if we always need the `count` addition here and below? Did you get this somewhere out of the convention?

Reply:
I'm adding `count` as the unit here https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html#abbreviations. Maybe it's not needed here?

Reply:
I would skip it. We mainly use it if we have also avg, sum etc. in parallel but it's not the case here.

Reply:
Oh `count` is not the unit... OK! Will remove it!! Thanks!